dashdotdb/services/serviceslometrics.py

```diff
@@ -8,19 +8,32 @@
 from dashdotdb.models.dashdotdb import Namespace
 from dashdotdb.models.dashdotdb import ServiceSLO
 from dashdotdb.models.dashdotdb import SLIType
+from dashdotdb.models.dashdotdb import SLODoc
 from dashdotdb.services import DataTypes
 from dashdotdb.controllers.token import (TOKEN_NOT_FOUND_CODE,
                                          TOKEN_NOT_FOUND_MSG)
 
 
+class ServiceSLOMetricsInput:
+    def __init__(self):
```

Comment:
Is the expectation that someone would create this object and then set values, rather than allowing values to be passed in like `ServiceSLOMetricsInput('cluster', 'namespace', ...)`?

Reply:
Thanks, I always appreciate your perspectives and input.

In the particular case of the `ServiceSLOMetrics.__init__()` function, sometimes `cluster`, `namespace`, etc. are expected be expected to be `None`. So I think I'm content with this part of the code as-is.

Next time I run into a problem like this I might just depend on `# pylint: disable=too-many-arguments` though. 🙂